types/three/examples/jsm/libs/lil-gui.module.min.d.ts

```diff
@@ -1 +1,609 @@
+     */
+    constructor({
+        parent,
+        autoPlace,
+        container,
+        width,
+        title,
+        injectStyles,
+        touchStyles,
+    }?: {
+        autoPlace?: boolean;
+        container?: HTMLElement;
+        width?: number;
+        title?: string;
+        injectStyles?: boolean;
+        touchStyles?: number;
+        parent?: GUI;
+    });
+    /**
+     * The GUI containing this folder, or `undefined` if this is the root GUI.
+     */
+    parent: GUI;
+    /**
+     * The top level GUI containing this folder, or `this` if this is the root GUI.
+     */
+    root: GUI;
+    /**
+     * The list of controllers and folders contained by this GUI.
+     */
+    children: Array<GUI | Controller>;
+    /**
+     * The list of controllers contained by this GUI.
+     */
+    controllers: Controller[];
+    /**
+     * The list of folders contained by this GUI.
+     */
+    folders: GUI[];
+    /**
+     * Used to determine if the GUI is closed. Use `gui.open()` or `gui.close()` to change this.
+     */
+    _closed: boolean;
+    /**
+     * Used to determine if the GUI is hidden. Use `gui.show()` or `gui.hide()` to change this.
+     */
+    _hidden: boolean;
+    /**
+     * The outermost container element.
+     */
+    domElement: HTMLElement;
+    /**
+     * The DOM element that contains the title.
+     */
+    $title: HTMLElement;
+    /**
+     * The DOM element that contains children.
+     */
+    $children: HTMLElement;
+    /**
+     * Adds a controller to the GUI, inferring controller type using the `typeof` operator.
+     * @example
+     * gui.add( object, 'property' );
+     * gui.add( object, 'number', 0, 100, 1 );
+     * gui.add( object, 'options', [ 1, 2, 3 ] );
+     *
+     * @param object The object the controller will modify.
+     * @param property Name of the property to control.
+     * @param [$1] Minimum value for number controllers, or the set of
+     * selectable values for a dropdown.
+     * @param [max] Maximum value for number controllers.
+     * @param [step] Step value for number controllers.
+     */
+    add<T, K extends keyof T>(
+        object: T,
+        property: K,
+        options: ReadonlyArray<T[K]> | Record<string, T[K]>,
+    ): OptionController<T, K>;
+    add<T, K extends KeyToValueOfType<T, number>>(
+        object: T,
+        property: K,
+        min?: number,
+        max?: number,
+        step?: number,
+    ): NumberController<T, K>;
+    add<T, K extends KeyToValueOfType<T, boolean>>(object: T, property: K, options?: never): BooleanController<T, K>;
+    add<T, K extends KeyToValueOfType<T, string>>(object: T, property: K, options?: never): StringController<T, K>;
+    add<T, K extends KeyToValueOfType<T, (this: T) => void>>(
+        object: T,
+        property: K,
+        options?: never,
+    ): FunctionController<T, K>;
+    /**
+     * Adds a color controller to the GUI.
+     * @example
+     * params = {
+     * 	cssColor: '#ff00ff',
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/three/examples/jsm/libs/lil-gui.module.min.d.ts;
  Line: 130;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.